src/VaultSharp/V1/SecretsEngines/PKI/SignCertificateRequestOptions.cs

```diff
@@ -0,0 +1,27 @@
+using Newtonsoft.Json;
+
+namespace VaultSharp.V1.SecretsEngines.PKI
+{
+    /// <summary>
+    /// Represents the Certificate credentials request options.
+    /// </summary>
+    public class SignCertificatesRequestOptions : AbstractCertificateRequestOptions
+    {
+
+        /// <summary>
+        /// <para>[required]</para>
+        ///  Specifies the PEM-encoded CSR
+        /// </summary>
+        /// <value>
+        ///  Encoded CSR.
+        /// </value>
+        [JsonProperty("csr")]
+        public string Csr { get; set; }
+
+        /// <summary>
```

Comment:
remove this. redundant

Reply:
Yes you right. Removed Abstract class and created separate classes for both endpoints so this one is fixed too.